compiler/parser/parser.cpp

```diff
@@ -882,6 +952,8 @@ static ModuleSymbol* uASTParseFile(const char* fileName,
     INT_FATAL("compiler library context not initialized");
   }
 
+  gContext->setErrorHandler(&uASTParseFileErrorHandler);
```

Comment:
IMO we should do this when we set up gContext so we have this set up across the whole compiler (which is only relevant when we go beyond what we have here).

Reply:
I'll add a TODO